resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/rest/kubernetes/v2/ResourceStagingServiceRetrofit.scala

```diff
@@ -37,6 +37,10 @@ private[spark] trait ResourceStagingServiceRetrofit {
 
   @Streaming
   @retrofit2.http.GET("/api/v0/resources/{resourceId}")
```

Comment:
This is V0 of this particular API but version 2 of submitting to Kubernetes in general.